irohad/network/sync_gate.hpp

```diff
@@ -0,0 +1,24 @@
+//
+// Created by Takumi Yamashita on 2017/07/11.
+//
+
+#ifndef IROHA_NETWORK_SYNCGATE_HPP
+#define IROHA_NETWORK_SYNCGATE_HPP
+namespace iroha {
+  namespace network {
+
+    /**
+     * Interface for downloading blocks from a network
+     */
+    class SyncGate {
```

Comment:
OrderingService
OrderingClient <-> OrderingGate
Consensus
ConsensusClient <-> ConsensusGate
Synchronization
SyncClient <-> SyncGate
is better. I think.